scripts/zones/Mount_Zhayolm/mobs/Cerberus.lua

```diff
@@ -37,5 +37,5 @@ end;
 -----------------------------------
 
 function onMobDespawn(mob)
-    mob:setRespawnTime((math.random(0,24)*3600)+172800); -- 48-72 hours proper 1 hour windows
+    mob:setRespawnTime(math.random(172800,259200)); -- 48-72 hours proper 1 hour windows
```

Comment:
1 or 3. But 3 would mean you get to look and see what each NM's mini-window sizes are and how many windows they get. Which is why 1 is an option. As much as my OCD loves consistency to death we can't take something that is correct and make it wrong so 2 is out.

(Back when I retail HNM'd, we called 48-72 "the window" and every 1 was the "mini window" it would actually pop at during that. I dunno what terms the kids use today)